aeloggingstackdriver.go

```diff
@@ -203,6 +203,28 @@ func logFromContext(ctx context.Context, sev logtypepb.LogSeverity, format strin
 	}
 }
 
+func AddLoggingLabels(log Logging, labels map[string]string) error {
+	levelLogger, ok := log.(*LevelLogger)
+	if !ok {
+		return fmt.Errorf("failed to add log labels, log needs to be a level logger")
+	}
+	stackdriverLogger, ok := levelLogger.wrappedLogger.(stackdriverLogging)
+	if !ok {
+		return fmt.Errorf("failed to add log labels, wrapped log needs to be a stackdriver logger", )
```

Comment:
trailing ','. Suppose one could just use errors.New() if not interpolating anything.

Reply:
Good spot. That was all leftover from some debug logs I had been doing :D Fixed it